tests/test_frame.py

```diff
@@ -2467,3 +2468,19 @@ def test_astype() -> None:
 
     states = pd.DataFrame({"population": population, "area": area})
     check(assert_type(states.astype(object), pd.DataFrame), pd.DataFrame, object)
+
+
+def test_xs_frame_new() -> None:
+    d = {
+        "num_legs": [4, 4, 2, 2],
+        "num_wings": [0, 0, 2, 2],
+        "class": ["mammal", "mammal", "mammal", "bird"],
+        "animal": ["cat", "dog", "bat", "penguin"],
+        "locomotion": ["walks", "walks", "flies", "walks"],
+    }
+    df = pd.DataFrame(data=d)
+    df = df.set_index(["class", "animal", "locomotion"])
+    s1 = df.xs(("mammal"), axis=0)
+    s2 = df.xs(("mammal"), axis=1)
```

Comment:
should be `s1 = df.xs("num_wings", axis=1)`

Reply:
yes I did that before and it gave a `Run Time Error`

Reply:
possibly because you had the extra parentheses.  I tried it and it worked

Reply:
i just pushed a commit regarding it can you check it